### PR TITLE
Improve readability and make code more intuitive, added zh_TW language

### PR DESCRIPTION
**Pull request by TWME-TW**
Improve readability and make code more intuitive

### CODE REVIEW

**Comment by TWME-TW** [comment]
I don't know why there are still 48 Commits, there should be only 5.

**Review by MenschenToaster — APPROVED** [review]
I think the changes are pretty unnecessary, but I think its okay for now. Also, thanks for translating.

In the future, I want the language system to change to provide per player translations. That means, if player abc has the language de_DE in their client, the server should use de_DE for translating. (Optional of course. The current could still be used)

But for now this is fine. Can this be merged?

And the amounts of commits probably exists because I just squashed your last pull request into one commit. And your repository has 50 more commits than mine.

**Comment by MenschenToaster** [comment]
And I would prefer if there are no major future changes done to the language file. Because if this is released, It's going to get ugly if every update breaks everything again.

**Comment by TWME-TW** [comment]
Can be merged, this will be the last format change (unless typo,like Glow Berries --> Cave Vines)

**Comment by TWME-TW** [comment]
> And I would prefer if there are no major future changes done to the language file. Because if this is released, It's going to get ugly if every update breaks everything again.

Maybe the file name will change if automatic language change is to be supported in the future

**Review by MenschenToaster — CHANGES_REQUESTED** [review]
Could you please remove the new listener and language class again? I think it's better if this gets merged first and the new language system gets added in the future.(If you don't mind, I would prefer if I implemented the new system). Since this PR creates syntax changes to the language file, I want this to be merged soon, so I can create a release.

File: src/main/resources/language.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ flower:
   FLOWERING_AZALEA: "§2Flowering Azalea"
   AZALEA_LEAVES: "§2Azalea Leaves"
   FLOWERING_AZALEA_LEAVES: "§2Flowering Azalea Leaves"
-  GLOW_BERRIES: "§2Glow Berries"
+  CAVE_VINES: "§2Cave Vines"
```
Comment (MenschenToaster) [inline src/main/resources/language.yml:104]: Is cave vines here really a better name? The icon is glow berries, and the vines are generally referred to as glow berries. Cave Vines is only the actual placed block. 

The community generally knows the block as glow berries, so we should keep that name

**Review by MenschenToaster — CHANGES_REQUESTED** [review]

**Comment by TWME-TW** [comment]
@MenschenToaster finished

**Comment by TWME-TW** [comment]
> Could you please remove the new listener and language class again? I think it's better if this gets merged first and the new language system gets added in the future.(If you don't mind, I would prefer if I implemented the new system). Since this PR creates syntax changes to the language file, I want this to be merged soon, so I can create a release.

Ok "Automatic language" is up to you to write(I am still in the groping stage)(I'm not quite sure how to say this in English, so maybe it's weird?)

**Review by MenschenToaster — APPROVED** [review]

**Pull request merged**